deploy/cloudbuild-release-lts.yaml

```diff
@@ -7,52 +7,15 @@ steps:
     args:
     - 'build'
     - '-t'
-    - 'gcr.io/$PROJECT_ID/build_deps:latest-lts'
-    - '--cache-from'
-    - 'gcr.io/k8s-skaffold/build_deps:latest-lts'
-    - '-f'
-    - 'deploy/skaffold/Dockerfile.deps.lts'
-    - '.'
-
-# Grab secret credentials from gcp bucket
-  - name: gcr.io/cloud-builders/gcloud
-    entrypoint: 'bash'
-    args: ['deploy/setup-secret.sh','-p', $PROJECT_ID]
-
-# Build and tag skaffold builder
-  - name: 'gcr.io/cloud-builders/docker'
-    args:
-    - 'build'
-    - '--cache-from'
-    - 'gcr.io/$PROJECT_ID/skaffold-builder:latest'
+    - 'gcr.io/$PROJECT_ID/skaffold:$TAG_NAME-lts'
     - '-t'
-    - 'gcr.io/$PROJECT_ID/skaffold-builder:latest'
+    - 'gcr.io/$PROJECT_ID/skaffold:$_SCANNING_MARKER-lts'
+    - '-t'
+    - 'us-east1-docker.pkg.dev/$PROJECT_ID/scanning/skaffold:$TAG_NAME-lts'
```

Comment:
This means, artifact registry scanner will scan all binary deps, probably we want to do it anyway. Just double check this is intended behavior?

Reply:
oh.. seems that our internal scanner is already doing this may be we don't need to push the image to ar anymore

Reply:
Okay, I removed AR. I wasn't sure why that was there but didn't want to change it...